ee/dtr/user/manage-images/sign-images/index.md

```diff
@@ -63,13 +62,13 @@ Repeat passphrase for new jeff key with ID a453196:
 Successfully imported key from key.pem
 ```
 
-### Initialize the trust metadata and add the Public Key
+### Initialize the trust metadata and add the user's public certificate
 
-Next, we need to initiate trust metadata for a DTR repository. If you have not
+Next,initiate trust metadata for a DTR repository. If you have not
 done so already, navigate to the **DTR web UI**, and create a repository for
-your image. In this example we've created the `prod/nginx` repository.
+your image. This example uses the `nginx` repository in the `prod` namespace.
 
-As part of initiating the repository, we will add the public key of the UCP User
+As part of initiating the repository, add the public key of the UCP user
```

Comment:
Nit. I would reword slightly "As part of initiating the repository, the public key of the UCP user will need to be added to the Notary server for this repository as a signer."